src/Utilities.js

```diff
@@ -64,3 +66,23 @@ export const getLookupConfigItems = (propertyTemplate) => {
 
   return templateConfigItems
 }
+
+/**
+ * Loads N3 into a dataset.
+ * @param {string} data that is the N3
+ * @return {Promise<rdf.Dataset>} a promise that resolves to the loaded dataset
+ */
+export const loadN3 = data => new Promise((resolve) => {
```

Comment:
Is this only, and will this only be, used in the test suite? If so, would you consider moving this to a new helper file in `__tests__/`? I think we have 1-2 examples of those in there.

Reply:
It currently is used only once. However, I anticipate that @jcoyne will use it in Trellis loading, as he has a similar method in https://github.com/LD4P/sinopia_editor/pull/808

Reply:
okie dokie. let's leave it here then!

Reply:
maybe add a comment re what you discussed, so we can take the knowledge forward:  ` * expect this to be used in Trellis loading, but currently only used for tests`  as that may help with future refactoring?